Core/Core/Api/GraphQL/Client.GraphqlCleintOperations/Client.BranchOperations.cs

```diff
@@ -160,6 +160,57 @@ public Task<Branch> BranchGet(string streamId, string branchName, int commitsLim
     return res.stream.branch;
   }
 
+  /// <summary>
+  /// Gets a given model from a project.
+  /// </summary>
+  /// <param name="cancellationToken"></param>
+  /// <param name="projectId">Id of the project to get the model from</param>
+  /// <param name="modelId">Id of the model</param>
+  /// <returns></returns>
+  public async Task<Branch> ModelGet(
```

Comment:
Cancellation token as last parameter
Dev has been updated with this change propagated to others

Reply:
Resolved in my pr #2984